src/components/InputBook.js

```diff
@@ -36,28 +37,31 @@ const InputBook = () => {
   };
   return (
     <>
-      <h3>ADD NEW BOOK</h3>
-      <form action="submit" className="submit">
-        <input
-          type="text"
-          className="title"
-          placeholder="Book title"
-          value={data.title}
-          name="title"
-          onChange={handleChange}
-        />
-        <input
-          type="text"
-          className="author"
-          placeholder="Author"
-          name="author"
-          value={data.author}
-          onChange={handleChange}
-        />
-        <button type="button" className="addBook" onClick={handleSubmit}>
-          Add Book
-        </button>
-      </form>
+      <hr className="form-hr" />
+      <div className="add-book">
+        <h3>ADD NEW BOOK</h3>
+        <form action="submit" className="submit">
```

Comment:
Please this form should take the full width of the ùain section just like it's upper component and inputs should fill the space 

![image (16)](https://github.com/ragangithub/bookstore-react/assets/59707859/500f9a20-2a25-4ee8-9eba-d1f13a941936)
![image (17)](https://github.com/ragangithub/bookstore-react/assets/59707859/72c7f7ac-cebc-4f61-88ca-cca4d3d95834)